mitosheet/mitosheet/streamlit/v1/spreadsheet.py

```diff
@@ -122,6 +122,28 @@ def __init__(self, code: str, code_options: Optional[CodeOptions], fully_paramet
     def param_metadata(self) -> List[ParamMetadata]:
         return self.__param_metadata
     
+    def to_json(self) -> str:
+        return json.dumps({
+            'code': self.__code,
```

Comment:
The one thing we should def add here is a version - so that if we change the format. 

I think let's:
1. Add a `self.mito_analysis_version = 1` variable to the class
2. Include this in the JSON. 

This will protect us in case we need to upgrade / change this format over time.